lib/matplotlib/legend.py

```diff
@@ -857,10 +862,10 @@ def get_texts(self):
         r"""Return the list of `~.text.Text`\s in the legend."""
         return silent_list('Text', self.texts)
 
-    def set_title(self, title, prop=None):
+    def set_title(self, title, align=None, prop=None):
```

Comment:
Please add this at the end of the list as otherwise it may break people's code if they are not using keyword arguments. (Possibly add it as a keyword only argument, so 
```suggestion
    def set_title(self, title, prop=None, *, align=None):
```
)

Reply:
Sure, I just made the changes!